docs/guides/rule-evaluations-via-query-frontend.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Rule evaluations via query frontend"
+linkTitle: "Rule evaluations via query frontend"
+weight: 10
+slug: rule-evalutions-via-query-frontend
+---
+
+This guide explains how to configure the Ruler to evaluate rules via Query Frontends instead of the Ingesters and the pros and cons of rule evaluation via Query Frontend.
+
+## How to enable
+
+By default, the Ruler queries to the Ingesters for evaluating rules (alerting rules or recording rules). If you have set `-ruler.frontend-address` then the Ruler query to the Query Frontend for evaluation rules.
+The address should be the gRPC listen address in host:port format.
+
+You can configure via args:
+```
+-ruler.frontend-address=query-frontend.svc.cluster.local:9095
+```
+And via yaml:
+```yaml
+ruler:
+  frontend_address: query-frontend.svc.cluster.local:9095
+```
+
+In addition, you can configure gRPC client (Ruler -> Query Frontend) config via args:
```

Comment:
I think it is unnecessary to list all configuration flags. This adds extra complexity for users who read this doc as I expect that most of the case default values should be good enough. 

Is it possible to use a link to redirect to the frontend client configurations?